packages/analytics-browser/src/config.ts

```diff
@@ -246,8 +246,9 @@ export const createEventsStorage = async (overrides?: BrowserOptions): Promise<S
   // If storageProvider is implicitly undefined `{ }`
   // then storageProvider is LocalStorage
   // Otherwise storageProvider is overriden
+  const loggerProvider = overrides && overrides.loggerProvider;
   if (!hasStorageProviderProperty || overrides.storageProvider) {
-    for (const storage of [overrides?.storageProvider, new LocalStorage<Event[]>()]) {
+    for (const storage of [overrides?.storageProvider, new LocalStorage<Event[]>({ loggerProvider: loggerProvider })]) {
```

Comment:
nit: i think shorthand is prefered in eslint
```suggestion
    for (const storage of [overrides?.storageProvider, new LocalStorage<Event[]>({ loggerProvider })]) {
```